syntax = "proto3";

package com.example.grpc;

option java_package = "com.example.grpc";
option java_outer_classname = "HelloWorldProto";
option java_multiple_files = true;

service Greeter {
  rpc SayHello (HelloRequest) returns (HelloReply) {}
  rpc StreamHello (HelloRequest) returns (stream HelloReply) {}
}

message HelloRequest {
  string name = 1;
  string language = 2;
}

message HelloReply {
  string message = 1;
  string language = 2;
}
